src/init.cpp

```diff
@@ -291,6 +291,7 @@ void Shutdown()
     globalVerifyHandle.reset();
     ECC_Stop();
     LogPrintf("%s: done\n", __func__);
+    async_logging::Shutdown();
```

Comment:
oh - why do in both? if you are just concerned about capturing all logs, see my comment on the Shutdown() method below